### PR TITLE
build/electrical cleanups

### PR DESCRIPTION
**Pull request by shiftingtech**
just fixing some case sensitivity issues, and removed one duplicate file

### CODE REVIEW

File: build/electrical/V1_octopus_wiring.md
Comment (HumbleDeer) [inline build/electrical/V1_octopus_wiring.md:1]: How was this file a duplicate? What is it a duplicate of?

Comment (shiftingtech) [reply build/electrical/V1_octopus_wiring.md:1]: V1_octopus_wiring.md and v1_octopus_wiring.md both existed.

Comment (shiftingtech) [reply build/electrical/V1_octopus_wiring.md:1]: I feel like these changes got rolled up into something else that actually went through though.

Comment (HumbleDeer) [reply build/electrical/V1_octopus_wiring.md:1]: Possibly? I can't find two files with the same name. 

Your branch is 11 commits behind the main branch. Please update your own branch before starting any edits. :)

**Pull request closed without merging**